apps/sertra/components/ServerDetail.module.css

```diff
@@ -0,0 +1,354 @@
+  line-height: 22px;
+
+  display: flex;
+  align-items: center;
+
+  color: #CCDDFE;
+  margin-top: 2em;
+  margin-bottom: 1em;
+
+}
+
+.descriptionContent {
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 1.5em 2em;
+  gap: 1em;
+
+  background: #242E48;
+
+  border: 0.125em solid #353E58;
+
+}
+
+.descriptionContentText {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 140%;
+
+  display: flex;
+  align-items: center;
+
+  color: #FFFFFF;
+
+}
+
+.modImg {
+  width: 64px;
+  height: 64px;
+
+  background: greenyellow;
+  border-radius: 0px;
+
+}
+
+.modName {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 19px;
+  display: flex;
+  align-items: center;
+
+  color: #FFFFFF;
+
+}
+
+.modDescription {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 140%;
+
+  display: flex;
+  align-items: center;
+
+  color: #CCDDFE;
+
+}
+
+.modRow {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 19px;
+
+  display: flex;
+  align-items: center;
+
+  color: #FFFFFF;
+  margin-top: 2em;
+  margin-bottom: 2em;
+
+}
+
+.modRowColumn {
+  display: flex;
+  flex-direction: column;
+}
+
+.modContentRow {
+  display: flex;
+  align-items: flex-start;
+}
+
+.modImg {
+  margin-right: 2em;
+}
+
+.modArtifacts {
+  display: flex;
+  flex-direction:row;
+  gap: 0.25em;
+}
+
+.artifact {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 0.25em 0.625em;
+  gap: 0.625em;
+
+  width: 4.25em;
+  height: 1.375em;
+
+  background: #353E58;
+
+}
+
+.artifactText {
+  width: 4em;
+  height: 1.16em;
+
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+
+  color: #CCDDFE;
+}
+
+.modName {
+  width: 35.25em;
+  height: 1.375em;
+
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 19px;
+  display: flex;
+  align-items: center;
+}
+
+.modDescription {
+  width: 40.285em;
+  height: 1.571em;
+
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 140%;
+
+  color: #CCDDFE;
+}
+
+.serverInfoContent {
+  gap: 1em;
+}
+
+.serverInfoRow {
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: row;
+  padding: 0.5em 1em;
+  gap: 1em;
+  margin-top: 0.2em;
+  margin-bottom: 0.2em;
+
+  width: 28.5em;
+
+
+  background: #242E48;
+
+  border: 2px solid #353E58;
+}
+
+.serverInfoColumn {
+  flex: 1;
+}
+
+.joinServerButton {
+
+  margin-top: 3.25em;
+  
+  display: flex;
+  align-self: end;
+  justify-content: center;
+  padding: 0.75em 1.875em;
+  gap: 0.75em;
+
+  width: 10.875em;
+  height: 2.6875em;
+  right: 24.375em;
+  top: 8.8125em;
+
+  background: #FFA96B;
+}
+
+.joinServerButtonText {
+  width: 5.1875em;
+  height: 1.1875em;
+
+  font-weight: 700;
+  color: #202941;
+}
+
+@media (max-width: 1199px) {
```

Comment:
IIRC the layout has a fixed width of 1140px so this seems a bit odd (might be what we want though). Anyways I guess we should unify the breakpoints/widths so all the pages seem consistent.

Reply:
I kind of just added this "mobile support" to try out how it would be done. I just replaced the fixed width in dev to 100% for the time I worked on it. I'm not sure if we want to add site scalability yet, but I'll add a task in the inbox.

Reply:
There's already a card for this (TS-522), which I intend to work on next. (Or I'm already working on some of it's subtasks, to be exact.) So I guess this is good "as is".